recipe/meta.yaml

```diff
@@ -25,23 +25,55 @@ requirements:
   host:
     - ncurses                # [not win]
     - readline               # [not win]
-  run:
-    - ncurses                # [not win]
-    - readline               # [not win]
 
-test:
-  commands:
-    - sqlite3 --version
-    - if not exist %PREFIX%\\Library\bin\sqlite3.dll exit 1    # [win]
-    - if not exist %PREFIX%\\Library\lib\sqlite3.lib exit 1    # [win]
-    - test -f $PREFIX/lib/libsqlite3.dylib                     # [osx]
-    - test -f $PREFIX/lib/libsqlite3.so                        # [linux]
-    - if not exist %PREFIX%\\Library\include\sqlite3.h exit 1  # [win]
-    - test -f $PREFIX/include/sqlite3.h                        # [not win]
+
+# By compiling with readline, the /bin/sqlite3 executable becomes
+# subject to the GPL
+# Therefore, we decide to split it to make it clear
+# That the other software is not.
```

Comment:
I don't think that this is strictly true, or at least it is open to legal interpretation. The GPL certainly claims that (dynamically) linking will make the joint program GPL.  However, this does not necessarily follow, as when you link the only part of the code that is used is the API, which is non-copyrightable.

Reply:
That is fair, but there is no compatible library that has a compatible API (yet) on conda-forge so that argument is rather weak.

I think that along with the dependency on `readline`, it is hard to argue against the fact that the final product, that arrives on the user's computer is not using GPL code.